pkg/cmd/server/start/start_master.go

```diff
@@ -84,6 +85,19 @@ func NewCommandStartMaster(basename string, out, errout io.Writer) (*cobra.Comma
 		Long:  fmt.Sprintf(masterLong, basename),
 		Run: func(c *cobra.Command, args []string) {
 			kcmdutil.CheckErr(options.Complete())
+
+			if options.PrintIP {
```

Comment:
Eventually this will go away too.  Why move it here?

Reply:
We still need it for like a billion test cases and tests.  Everything using start went to using start master, so I moved it here.